BlockBench.md

```diff
@@ -0,0 +1,121 @@
+# Blockbench To Minecraft Legends
+This article will walk you through how to add custom models and textures to Minecraft Legends using the popular free software Blockbench. Adding custom art content is essential to make your experience stand out and look different from the base game. For more information on using blockbench please visit the following link:
+
+https://www.blockbench.net/quickstart
+
+and then go to Minecraft: Bedrock Edition > Entity/model for some quick start information and a few tutorials. 
+
+```
+Important! 
+Be sure to only use the Bedrock Entity Project type when creating projects as the exporter only supports cube geometry.
+```
+
+In this Article you will learn the following:
+
+* How to install Blockbench
+
+* How to install the plugin
+
+* How to export models, animations and textures
+
+
+# Minecraft Legends Exporter Overview
+
+Minecraft Legends Exporter is a plugin for Blockbench to help with exporting models into a format that Minecraft Legends can use. This Article will cover how to install the plugin and how to export models, textures, animations as well as tips & best practices.
+
+# Download Blockbench
+
+To use this plugin, you need [Blockbench](https://www.blockbench.net/downloads)
+
+# Installing the plugin
+
+1.  Download the [Minecraft: Legends plugin](https://github.com/Mojang/legends-blockbench-plugin/releases)
+2.	Open Blockbench and using the menu bar select **File** > **Plugins**
+
+![](images/blockbench_doc/image01.png)
+
+3.	Select the load plugin icon (the icon between “Plugins” and the cloud icon). This will open a file dialog. From the file dialog, navigate to where you saved the downloaded plugin, select the plugin, and then select Open. 
```

Comment:
These steps kind of imply that users should try to open the downloaded zip file from the releases page (which of course is not right).  I think we should be very explicit about unzipping the downloaded release and pointing at the appropriate js file.

Reply:
Good call!